File: src/JavaScript/easy/string/problem1876.js
/**
 * @param {string} s
 * @return {number}
 */

var countGoodSubstrings = function (s) {
  let goodChars = [];
  for (let i = 0; i <= s.length; i++) {
    if (i + 3 <= s.length) {
      let ss = s.split("").splice(i, 3).join("");
      if(new Set(ss.split("")).size == 3){
        goodChars.push(ss)
      }
    }
  }
  return goodChars.length
};

// let s = "xyzzaz";
let s = "aababcabc"

countGoodSubstrings(s);

File: src/Python/easy/string/problem1876.py
class Solution:
    def countGoodSubstrings(self, s: str) -> int:
        goodChars = []
        for i in range(len(s)):
            if i + 3 <= len(s):
                ss = s[i:i + 3]
                if len(set(ss)) == 3:
                    goodChars.append(ss)
        return len(goodChars)




s = "xyzzaz"

solution = Solution()
solution.countGoodSubstrings(s)